src/loaders/jobs/taskfarmer/taskfarmer_task_mgr.py

```diff
@@ -50,6 +53,33 @@ class TFTaskManager:
     The task manager is responsible for managing the tasks of TaskFarmer jobs.
     """
 
+    @staticmethod
+    def _read_last_n_line(file_path, n=5):
```

Comment:
I'd use a package for this rather than rolling your own, https://pypi.org/project/file-read-backwards/ might be reasonable

Reply:
👍